<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormStatistic</class>
 <widget class="QWidget" name="FormStatistic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/*QHeaderView 
{
	font: bold 12pt &quot;Microsoft YaHei&quot;;
  	min-height:30px;
	background-color:#dbdbdb;
}*/

QHeaderView::section {
     background-color: qlineargradient( x1:0, y1:0, x2:0, y2:1,
                                   stop:0 #616161, 
                                   stop: 0.5 #505050,
                                   stop: 0.6 #434343, stop:1 #656565);

	font: bold 12pt &quot;Microsoft YaHei&quot;;
	color:white;
  	min-height:26px;
    border: 1px solid #6c6c6c;
}
QTableWidget
{
    font-size:11pt &quot;Microsoft YaHei&quot;;
    /*background-color: rgb(247,247,247);
	gridline-color: rgb(217, 217, 217);*/
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetToolBar" native="true">
     <property name="styleSheet">
      <string notr="true">*
{
	font: 12pt &quot;Microsoft YaHei UI&quot;;
}
/*
#widgetToolBar
{
	border:1px solid gray;
}*/</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="lbTotalPeriod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>总期数</string>
        </property>
        <property name="buddy">
         <cstring>leTotalPeriod</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leTotalPeriod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbTatisticPeriod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>统计期数</string>
        </property>
        <property name="buddy">
         <cstring>sbTatisticPeriod</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sbTatisticPeriod">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>72</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbTatisticMode">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>统计方式</string>
        </property>
        <property name="buddy">
         <cstring>cbStatisticMode</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbStatisticMode">
        <item>
         <property name="text">
          <string>统计热度</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>统计遗漏</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbTatisticRank">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>统计名次</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>9</number>
        </property>
        <item>
         <widget class="QCheckBox" name="cbrOne">
          <property name="text">
           <string>一</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbrTwo">
          <property name="text">
           <string>二</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbrThree">
          <property name="text">
           <string>三</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbrFour">
          <property name="text">
           <string>四</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbrFive">
          <property name="text">
           <string>五</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbrSix">
          <property name="text">
           <string>六</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbrSeven">
          <property name="text">
           <string>七</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbrEight">
          <property name="text">
           <string>八</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbrNine">
          <property name="text">
           <string>九</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbrTen">
          <property name="text">
           <string>十</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="twStatisticList">
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
